### PR TITLE
feat: document how to remove environments

### PR DESCRIPTION
**Pull request by obreitwi**
Rev: SRX-PLUW27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,3 +214,61 @@ The `"argocd"` field has a few subfields:
...
+
+- Remove the `<Stage>` (in the example below `development` or `development2`) kuberpult overlay from the service so that it does not get deployed anymore.
+- Merge the overlay removal(s), triggering a new release.
+
+#### In Kuberpult:
+
+- Remove environments from app (for each app/environment):
+![](./assets/img/remove_env/dots_menu.png)  
+![](./assets/img/remove_env/from_app.png)
+- Select environment(s) to remove
```
Comment (sven-urbanski-freiheit-com) [inline README.md:238]:
```suggestion
- Select environment(s) to remove. Note that you *cannot* remove all environments, one has to stay. If you want to remove a service entirely, see "Remove a service entirely"
```

File: README.md [hunk 2]
```diff
@@ -214,3 +214,61 @@ The `"argocd"` field has a few subfields:
...
+- Hit "prepare to undeploy"  
+  ![](./assets/img/undeploy/prepare.png)
+- Be sure to remove all locks specific to this service & hit apply  
+  ![](./assets/img/undeploy/locks.png)
+- Hit "Apply" & confirm  
+  ![](./assets/img/undeploy/apply.png)  
+- This creates an empty manifest that can be deployed like any other version.  
+  ![](./assets/img/undeploy/version.png)
+- Then, alternatively:
+  - Either Wait for the normal release trains to roll this out on all stages
```
Comment (sven-urbanski-freiheit-com) [inline README.md:266]:
```suggestion
  - Either wait for the normal release trains to roll this out on all stages
```

File: README.md [hunk 3]
```diff
@@ -214,3 +214,61 @@ The `"argocd"` field has a few subfields:
...
+  - Or Deploy the "undeploy version" to all stages manually
```
Comment (sven-urbanski-freiheit-com) [inline README.md:267]:
```suggestion
  - Or deploy the "undeploy version" to all stages manually
```

File: README.md [hunk 4]
```diff
@@ -214,3 +214,61 @@ The `"argocd"` field has a few subfields:
 The `"environmentGroup"` field is a string that defines which environment group the environment belongs to (Example: `Production` can be an environment group to group production environments in different countries).
 EnvironmentGroups are still in development. We'll update this section once they are ready.
 The goal of EnvironmentGroups is to make handling of many similar clusters easier. They will also work with Release Trains.
+
+---
+
+## Best practices
+
+### Remove individual stages from a service
```
Comment (sven-urbanski-freiheit-com) [inline README.md:222]: for consistency, please replace "stage" with "environment" everywhere

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

**Review by jaypontes — APPROVED** [review]

**Pull request merged**